media-video/qtav/qtav-1.11.0.recipe

```diff
@@ -0,0 +1,152 @@
+SUMMARY="A cross-platform multimedia framework"
+DESCRIPTION="
+QtAV is a multimedia playback library based on Qt and FFmpeg. \
```

Comment:
It is better to join DESCRIPTION and its first line:
```
DESCRIPTION="QtAV is a multimedia playback library based on Qt and FFmpeg. \
```
(For already existing recipes, sometimes we don't bother to respect that tiny rule, but for new recipes we should do so.)